src/shadowbox/test_action.sh

```diff
@@ -14,5 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-do_action shadowbox/server/build
+rm -rf $BUILD_DIR/js
```

Comment:
Same as before: only delete what this script creates

Reply:
Done.